src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -541,7 +541,7 @@ def load_arguments(self, _):
 
     with self.argument_context('vmss create', min_api='2017-03-30', arg_group='Network') as c:
         c.argument('public_ip_per_vm', action='store_true', help="Each VM instance will have a public ip. For security, you can use '--nsg' to apply appropriate rules")
-        c.argument('vm_domain_name', help="domain name of VM instances, once configured, the FQDN is 'vm<vm-index>.<vm-domain-name>.<..rest..>'")
+        c.argument('vm_domain_name', help="domain name of VM instances, once configured, the FQDN is `vm<vm-index>.<vm-domain-name>.<..rest..>`")
```

Comment:
@qwordy

Reply:
How do you send picture?

Reply:
> How do you send picture?

Create a screenshot and paste it into input box? I guess.
This is the way I used.